tests/dump_tests/module_tests/portchannel_test.py

```diff
@@ -0,0 +1,128 @@
+import json, os, sys
+import jsonpatch
+import unittest
+import pytest
+from deepdiff import DeepDiff
+from mock import patch
+from dump.helper import create_template_dict, sort_lists
+from dump.plugins.portchannel import Portchannel
+from dump.match_infra import MatchEngine, ConnectionPool
+from swsscommon.swsscommon import SonicV2Connector
+
+
+# Location for dedicated db's used for UT
+module_tests_path = os.path.dirname(__file__)
+dump_tests_path = os.path.join(module_tests_path, "../")
+tests_path = os.path.join(dump_tests_path, "../")
+dump_test_input = os.path.join(tests_path, "dump_input")
+port_files_path = os.path.join(dump_test_input, "portchannel")
+
+# Define the mock files to read from
+dedicated_dbs = {}
+dedicated_dbs['CONFIG_DB'] = os.path.join(port_files_path, "config_db.json") 
+dedicated_dbs['APPL_DB'] = os.path.join(port_files_path, "appl_db.json") 
+dedicated_dbs['ASIC_DB'] = os.path.join(port_files_path, "asic_db.json")
+dedicated_dbs['STATE_DB'] = os.path.join(port_files_path, "state_db.json")
+
+
+def populate_mock(db, db_names):
```

Comment:
this function seems to be used across the unit test cases, can it added in a way that it can be re-used/